### DISCUSSION TITLE
Name ("KDL")?

### DISCUSSION THREAD

**Discussion by enterprisey** [kdl-org/kdl#175] [Q&A] [answered]
First off, I love it. I will do my best to use this in everything going forward.

What's the story behind the name? "KDL" is sorta... nondescript.

**Comment by sthagen** [comment]
Skimming the repo page it seems it is Kat's Document Language and reading further it derives from SDL ... does this help?

**Comment by zkat** [comment, marked as answer]
Originally, it was indeed Kat's Document Language. Both a pun on SDLang, and a jab at TOML.

But then I felt awkward having my name on it so it's officially "KDL Document Language" now.

Hope that answers it!

**Reply by sthagen** [reply]
The forks still have the description with your name @zkat  - I always liked that, but will sync my fork with your new upstream info. Thanks for sharing the KDL with the rest of us and for offering your own and welcoming way of communicating - much appreciated!
